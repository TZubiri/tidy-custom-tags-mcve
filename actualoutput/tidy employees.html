line 9 column 5 - Error: <employee> is not recognized!
line 9 column 5 - Warning: discarding unexpected <employee>
line 10 column 7 - Error: <name> is not recognized!
line 10 column 7 - Warning: discarding unexpected <name>
line 10 column 26 - Warning: discarding unexpected </name>
line 11 column 7 - Error: <role> is not recognized!
line 11 column 7 - Warning: discarding unexpected <role>
line 11 column 29 - Warning: discarding unexpected </role>
line 12 column 7 - Error: <longevity> is not recognized!
line 12 column 7 - Warning: discarding unexpected <longevity>
line 12 column 36 - Warning: discarding unexpected </longevity>
line 13 column 7 - Error: <introduction> is not recognized!
line 13 column 7 - Warning: discarding unexpected <introduction>
line 13 column 155 - Warning: discarding unexpected </introduction>
Tidy found 34 warnings and 17 errors! Not all warnings/errors were shown.

This document has errors that must be fixed before
using HTML Tidy to generate a tidied up version.

About HTML Tidy: https://github.com/htacg/tidy-html5
Bug reports and comments: https://github.com/htacg/tidy-html5/issues
Official mailing list: https://lists.w3.org/Archives/Public/public-htacg/
Latest HTML specification: http://dev.w3.org/html5/spec-author-view/
Validate your HTML documents: http://validator.w3.org/nu/
Lobby your company to join the W3C: http://www.w3.org/Consortium

¿Le gustaría ver Tidy en un español correcto? Por favor considere
ayudarnos a localizar HTML Tidy. Para más detalles consulte
https://github.com/htacg/tidy-html5/blob/master/README/LOCALIZE.md
