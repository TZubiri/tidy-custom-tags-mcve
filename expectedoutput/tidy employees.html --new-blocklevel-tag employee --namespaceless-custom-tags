
line 10 column 7 - Warning: <name> is not recognized!
If this is a custom element you can use the '--new-blocklevel-tags name' option to let tidy know. To allow any custom tag you can use '--new-blocklevel-tags *', to read custom definitions from css you can use '--new-blocklevel-tags stylesheet.css'.
line 11 column 7 - Warning: <role> is not recognized!
line 12 column 7 - Warning: <longevity> is not recognized!
line 13 column 7 - Warning: <introduction> is not recognized!
line 14 column 7 - Warning: <impacts> is not recognized!
line 15 column 8 - Warning: <impact> is not recognized!

This document has errors that must be fixed before
using HTML Tidy to generate a tidied up version.

About HTML Tidy: https://github.com/htacg/tidy-html5
Bug reports and comments: https://github.com/htacg/tidy-html5/issues
Official mailing list: https://lists.w3.org/Archives/Public/public-htacg/
Latest HTML specification: http://dev.w3.org/html5/spec-author-view/
Validate your HTML documents: http://validator.w3.org/nu/
Lobby your company to join the W3C: http://www.w3.org/Consortium

¿Le gustaría ver Tidy en un español correcto? Por favor considere
ayudarnos a localizar HTML Tidy. Para más detalles consulte
https://github.com/htacg/tidy-html5/blob/master/README/LOCALIZE.md
