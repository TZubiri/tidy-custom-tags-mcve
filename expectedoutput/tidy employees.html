line 9 column 5 - Warning: <employee> is a custom tag, but it doesn't follow the [naming conventions](https://html.spec.whatwg.org/multipage/custom-elements.html#valid-custom-element-name) set by the w3c.
To disable this warning just ensure that you use lowercase letters and include 1 dash like so 'custom-tag'!
The best practice is to use the first part as a [namespace](https://en.wikipedia.org/wiki/Namespace) and the second part as the tag's name. like so 'namespace-employee'.
To disable this warning, you can use the --namespaceless-custom-tags, but beware, there is the risk of name collision if a browser assigns meaning to this tag.!

This document has errors that must be fixed before
using HTML Tidy to generate a tidied up version.

About HTML Tidy: https://github.com/htacg/tidy-html5
Bug reports and comments: https://github.com/htacg/tidy-html5/issues
Official mailing list: https://lists.w3.org/Archives/Public/public-htacg/
Latest HTML specification: http://dev.w3.org/html5/spec-author-view/
Validate your HTML documents: http://validator.w3.org/nu/
Lobby your company to join the W3C: http://www.w3.org/Consortium

¿Le gustaría ver Tidy en un español correcto? Por favor considere
ayudarnos a localizar HTML Tidy. Para más detalles consulte
https://github.com/htacg/tidy-html5/blob/master/README/LOCALIZE.md
